.clj-kondo/config/modules/config.edn

```diff
@@ -1969,8 +1969,10 @@
   {:team "Querying"
    :api  #{metabase-enterprise.transforms.api
            metabase-enterprise.transforms.core
-           metabase-enterprise.transforms.init}
-   :uses #{api
+           metabase-enterprise.transforms.init
+           metabase-enterprise.transforms.util}
```

Comment:
This is probably fine for now, but devex might want us to potempkin these through `core`